.github/CONTRIBUTING.md

```diff
@@ -1,23 +1,23 @@
-# nf-core/metaval: Contributing Guidelines
+# genomic-medicine-sweden/meta-val: Contributing Guidelines
 
 Hi there!
-Many thanks for taking an interest in improving nf-core/metaval.
+Many thanks for taking an interest in improving genomic-medicine-sweden/meta-val.
 
-We try to manage the required tasks for nf-core/metaval using GitHub issues, you probably came to this page when creating one.
+We try to manage the required tasks for genomic-medicine-sweden/meta-val using GitHub issues, you probably came to this page when creating one.
 Please use the pre-filled template to save time.
 
 However, don't be put off by this template - other more general issues and suggestions are welcome!
 Contributions to the code are even more welcome ;)
 
 > [!NOTE]
-> If you need help using or modifying nf-core/metaval then the best place to ask is on the nf-core Slack [#metaval](https://nfcore.slack.com/channels/metaval) channel ([join our Slack here](https://nf-co.re/join/slack)).
+> If you need help using or modifying genomic-medicine-sweden/meta-val then the best place to ask is on the nf-core Slack [#metaval](https://nfcore.slack.com/channels/metaval) channel ([join our Slack here](https://nf-co.re/join/slack)).
```

Comment:
This slack does not exist, so probably remove that?

Reply:
Done